lib/install/stimulus.rb

```diff
@@ -0,0 +1,12 @@
+say "Appending Stimulus setup code to #{Webpacker.config.source_entry_path}/application.js"
+append_to_file "#{Webpacker.config.source_entry_path}/application.js" do
+  "\n" + open("#{__dir__}/examples/stimulus/application.js").read
+end
+
+say "Creating controllers directory"
+directory "#{__dir__}/examples/stimulus/controllers", "#{Webpacker.config.source_entry_path}/controllers"
```

Comment:
We should create an `app/javascript/controllers/` directory here, not `app/javascript/packs/controllers/`. So use `Webpacker.config.source_path` instead of `Webpacker.config.source_entry_path`.

That's why https://github.com/rails/webpacker/pull/1244#discussion_r166323059 worked for me and not you.